specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
+
+As you can see, "Arial Narrow" has the same `lfWeight` and `lfItalic` values as "Arial", so it 
+has a different RBIZ family name to avoid ambiguity. "Arial Black" has a different RBIZ family name
+to avoid having more than two weights in the "Arial" family. By contrast, all of these fonts are in
+the same weight-stretch-style family.
+
+Nevertheless, the weight-stretch-style model is not open-ended. If two fonts have the same weight,
+stretch, and style but differ in some other way (e.g., optical size), they cannot be included in the
+same WWS font family. This brings us to the typographic font family model.
+
+## Typographic Font Family Model
+
+Unlike its predecessors, the typographic font family model is open-ended. It supports any number of axes
+of variation within a font family.
+
+If you think of font selection parameters as coordinates in a design space, the weight-stretch-style
+model defines a three-dimensional coordinate system with weight, stretch, and style as the axes. Each
+font in a WWS family must have a unique location defined by its coordinates along those three axes. To
+select a font, one specifies a WWS family name and weight, stretch, and style parameters.
+
+By contrast, the typographic font family model has an N-dimensional design space. A font designer can 
+define any number of design axes, each identified by a four-character _axis tag_. A given font’s location 
+in the N-dimensional design space is defined by an array of _axis values_, where each axis value comprises 
+an axis tag and a floating-point value. To select a font, one specifies a typographic family name and an 
+array of axis values (`DWRITE_FONT_AXIS_VALUE` structures).
+
+Although the number of font axes is open-ended, there are a few registered axes with standard meanings,
+and the weight, stretch, and style values can be mapped to registered axis values. `DWRITE_FONT_WEIGHT`
+can be mapped to a "wght" (`DWRITE_FONT_AXIS_TAG_WEIGHT`) axis value. `DWRITE_FONT_STRETCH` can be mapped
+to a "wdth" (`DWRITE_FONT_AXIS_TAG_WIDTH`) axis value. `DWRITE_FONT_STYLE` can be mapped to a combination
+of "ital" and "slnt" (`DWRITE_FONT_AXIS_TAG_ITALIC` and `DWRITE_FONT_AXIS_TAG_SLANT`) axis values. 
+
+Another registered axis is "opsz" (`DWRITE_FONT_AXIS_TAG_OPTICAL_SIZE`). An optical font family such as
+Sitka includes fonts that differ along the "opsz" axis, meaning they are designed to be used at different
+point sizes. The WWS font family model does not have an optical size axis, so the Sitka font family must
+be split into multiple WWS font families: "Sitka Small", "Sitka Text", "Sitka Subheading", and so on. Each
+WWS font family corresponds to a different optical size, and it's left to the user to specify the right
+WWS family name for a given font size. With the typographic font family model, the user can simply choose
+"Sitka", and the application can automatically set the "opsz" axis value based on the font size.
```

Comment:
How is "automatically" to be implemented? Would the application need to "rematch" the font for each size? Or would the application call font matching ones, then "opsz" would be implemented in rendering calls? 

It's hard for me to think of `opsz` as part of font matching, unless explicitly specified as search criterion.